hl7/4.2/modules/ROOT/pages/index.adoc

```diff
@@ -3,104 +3,55 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-Anypoint Connector for HL7 EDI (HL7 EDI Connector) lets you convert HL7 ER7 messages to and from DataWeave-compatible representations using lists and maps.
+Anypoint Connector HL7 EDI (HL7 EDI Connector) is a collection of features that facilitate integration with healthcare systems by providing the tools needed to easily create, read and transform HL7 version v2.x messages within Mule. Health Level Seven International (HL7) is a standards development organization that defines a set of standards for the transport and communication of healthcare information.
 
-Release Notes: https://docs.mulesoft.com/release-notes/connector/hl7-connector-release-notes-mule-4[Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/com.mulesoft.connectors/mule-hl7-extension[HL7 EDI Connector]
+For compatibility information and fixed issues, see the HL7 EDI Connector Release Notes.
 
-== Prerequisites
-
-This document assumes that you are familiar with HL7, Mule, Anypoint
-Connectors, Anypoint Studio, Mule flows, and Mule Global Elements.
-
-See the Release Notes for compatibility information. A link to the
-release notes is listed in the See Also section of this document.
-
-You must purchase a license for the HL7 EDI Connector to use it in a production environment.
-
-== POM File Information
-
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>com.mulesoft.connectors</groupId>
-  <artifactId>mule-hl7-extension</artifactId>
-  <version>4.0.3</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
-
-Replace version 4.0.3 for the version you are using. To specify a version, view Anypoint Exchange and click *Dependency Snippets*.
-
-== What's New in this Connector
-
-The data structures used by the HL7 EDI Connector 4.0.0 differ from
-those in the 3.x version used with Mule 3.x.
-
-In the 3.x version of the connector, composite components are effectively in-lined
-at the point of use, so that each individual element
-nested within the composite component is given a key based on the
-containing segment and referenced directly from the map
-representing the segment.
+Standard schema definitions are included in HL7 EDI Connector for
```

Comment:
Here, instead of listing all of these (and having to maintain in two places), should you just link to the schemas file?

Reply:
It seems to make sense, but I left it because I don't know how often these change. I will consider with the PM.